biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -57,7 +59,8 @@ subroutine mortality_rates( cohort_in,bc_in,cmort,hmort,bmort,frmort )
     real(r8),intent(out) :: frmort ! freezing stress mortality
 
     real(r8) :: frac  ! relativised stored carbohydrate
-    real(r8) :: b_leaf ! target leaf biomass kgC
+    real(r8) :: leaf_c_target      ! target leaf biomass kgC
```

Comment:
so again to clear up some possible confusion, we seem to still be interchanging b_leaf with leaf_c in some places, is that correct?  Or should we replace all previous b_leaf with leaf_c to represent leaf biomass?

Reply:
these naming conventions should definitely be cleared up.  I'm pretty sure that anywhere in the code you see "b_leaf", it actually is referring to the target leaf biomass, ie the value of the pool when it is on allometry and fully flushed.

Reply:
OK got it